src/main/java/com/artipie/api/ManageStorageAliases.java

```diff
@@ -0,0 +1,216 @@
+            for (final YamlNode node : storages.get().keys()) {
+                final String name = node.asScalar().value();
+                builder = builder.add(name, storages.get().yamlMapping(name));
+            }
+        }
+        builder = builder.add(alias, new Json2Yaml().apply(info.toString()));
+        this.blsto.save(
+            this.settingKey().orElse(
+                this.key.<Key>map(val -> new Key.From(val, ManageStorageAliases.YAML))
+                    .orElse(ManageStorageAliases.YAML)
+            ),
+            Yaml.createYamlMappingBuilder().add(ManageStorageAliases.STORAGES_NODE, builder.build())
+                .build().toString().getBytes(StandardCharsets.UTF_8)
+        );
+    }
+
+    @Override
+    public void remove(final String alias) {
+        final Optional<YamlMapping> storages = this.storages();
+        if (storages.isPresent() && storages.get().value(alias) != null) {
+            YamlMappingBuilder builder = Yaml.createYamlMappingBuilder();
+            for (final YamlNode node : storages.get().keys()) {
+                final String name = node.asScalar().value();
+                if (!alias.equals(name)) {
+                    builder = builder.add(name, storages.get().yamlMapping(name));
+                }
+            }
+            this.blsto.save(
+                this.settingKey().get(),
+                Yaml.createYamlMappingBuilder()
+                    .add(ManageStorageAliases.STORAGES_NODE, builder.build())
+                    .build().toString().getBytes(StandardCharsets.UTF_8)
+            );
+            return;
+        }
+        throw new IllegalStateException(String.format("Storage alias %s does not exist", alias));
+    }
+
+    /**
+     * Returns storages yaml mapping if found.
+     * @return Settings storages yaml
+     */
+    private Optional<YamlMapping> storages() {
+        final Optional<Key> stng = this.settingKey();
+        Optional<YamlMapping> res = Optional.empty();
+        if (stng.isPresent()) {
+            try {
+                res = Optional.ofNullable(
+                    Yaml.createYamlInput(
+                        new String(this.blsto.value(stng.get()), StandardCharsets.UTF_8)
+                    ).readYamlMapping().yamlMapping(ManageStorageAliases.STORAGES_NODE)
+                );
+            } catch (final IOException err) {
+                throw new UncheckedIOException(err);
+            }
+        }
+        return res;
+    }
+
+    /**
+     * Finds storages settings key.
+     * @return The key if found
+     */
+    private Optional<Key> settingKey() {
+        Optional<Key> res = Optional.of(
+            this.key.<Key>map(val -> new Key.From(val, ManageStorageAliases.YAML))
+                .orElse(ManageStorageAliases.YAML)
+        );
+        if (!this.blsto.exists(res.get())) {
+            final String yml = String.format("%s.yml", ManageStorageAliases.FILE_NAME);
+            res = Optional.of(
```

Comment:
It looks like the same above

Reply:
it's not the same: above we `yaml` extension, here we use `yml`

Reply:
Ok, I didn't notice an extension